.github/workflows/job-precommit-python-runners-portability-samza.yml

```diff
@@ -0,0 +1,81 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Test for Python Precommit Runners Portability Samza
+
+
+name: Python Precommit Runners Portability Samza
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+    paths: ['sdks/python/apache_beam/runners/portability/samza_runner_test.py']
```

Comment:
Please remove this, postcommits shouldn't be gated on paths

Reply:
Hi @damccorm This is a precommit job, should be applied the same change as well?

Reply:
@damccorm just to validate, should I apply the change considering that this is a pre-commit, not a post-commit?

Reply:
is there any comment on this?

Reply:
Sorry for the slow response; you're right, this is fine as is.